optimum/exporters/tasks.py

```diff
@@ -637,9 +637,11 @@ def determine_framework(model: str, framework: str = None) -> str:
             4. If could not infer, use available framework in environment, with priority given to PyTorch.
 
         Args:
-            model (`str`):
+            model_name (`str`):
                 The name of the model to export.
-            framework (`str`, *optional*):
+            subfolder (`str`, *optional*, defaults to `""`):
+                The subfolder where the model is stored.
```

Comment:
```suggestion
                In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
                specify the folder name here.
```